File: src/main/java/se/gigurra/gat/util/BufferUtil.java
package se.gigurra.gat.util;

import java.nio.FloatBuffer;
import java.nio.IntBuffer;

import com.jogamp.common.nio.Buffers;

public class BufferUtil {

	public static FloatBuffer createBuffer(final float... floats) {
		return Buffers.newDirectFloatBuffer(floats);
	}

	public static IntBuffer createBuffer(final int... ints) {
		return Buffers.newDirectIntBuffer(ints);
	}

}

File: src/main/scala/se/gigurra/gat/util/BufferUtil.scala
package se.gigurra.gat.util;

import java.nio.FloatBuffer
import java.nio.IntBuffer

import com.jogamp.common.nio.Buffers

object BufferUtil {

  def createBuffer(floats: Array[Float]): FloatBuffer = {
    Buffers.newDirectFloatBuffer(floats);
  }

  def createBuffer(ints: Array[Int]): IntBuffer = {
    Buffers.newDirectIntBuffer(ints);
  }

}
